toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -140,6 +140,18 @@ func NewOverlayMountPoint(chrootDir, source, target, lowerDir, upperDir, workDir
 	return
 }
 
+func (m *MountPoint) GetSource() string {
```

Comment:
Is this being used for anything?

Reply:
It's being using mic (different PR not published yet).

Reply:
Move this to the other change then?

Reply:
Once the other PR is published, please also link both of them with #XXXX mentions, so it's easier to see the connection between the changes.

Reply:
My plane is to make check-ins to the lower level layers first (IsoMaker) and then follow-up with the higher level layers.
These changes are library changes. Applications that depend on them will come in separate PRs.
I'll revert this change since it's harder to fit within the 'allow IsoMaker lib to take in a config struct instead of a config file.'

Reply:
Reverted.